packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -323,6 +381,58 @@ export class TabNav {
   //
   //--------------------------------------------------------------------------
 
+  private findVisibleTabTitleIndex = (tabTitles: NodeListOf<Element>, isNext: boolean): number => {
+    const mobilePageWidth = this.el.getBoundingClientRect().width;
```

Comment:
This const is a bit confusing because this is the component's width and not a page width.

Reply:
Can you use `this.el.clientWidth` or do you need to call `getBoundingClientRect()`?